### PR TITLE
workflows/tests: query GraphQL API with `curl`

### PR DESCRIPTION
**Pull request by carlocab**
The errors we've been constantly getting from querying the GraphQL API
might be due to a bug in actions/github-script. See
actions/github-script#230.

To work around this, let's just query the API using ~~`gh`~~`curl` instead.

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -54,17 +53,32 @@ jobs:
...
+              repository(owner: $owner, name: $name) {
+                pullRequests(last: 100, states: OPEN, labels: [$label]) {
+                  totalCount
+                }
+              }
+            }
+          '
+
+          OWNER='${{ github.repository_owner }}'
+          REPO_NAME=$(echo '${{ github.repository }}' | cut -f 2 -d '/')
```
Comment (carlocab) [inline .github/workflows/tests.yml:70]: We could use `github.event.repository.name` here (see https://stackoverflow.com/a/62805013), but I couldn't find any information on it so I'd rather not rely on that.

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -54,17 +53,32 @@ jobs:
...
+              repository(owner: $owner, name: $name) {
+                pullRequests(last: 100, states: OPEN, labels: [$label]) {
+                  totalCount
+                }
+              }
+            }
+          '
+
+          OWNER='${{ github.repository_owner }}'
+          REPO_NAME="$(echo '${{ github.repository }}' | cut -f 2 -d '/')"
```
Comment (carlocab) [inline .github/workflows/tests.yml:70]: We could use `github.event.repository.name` here (see https://stackoverflow.com/a/62805013), but I couldn't find any documentation on it so I'd rather not rely on that.

**Review by MikeMcQuaid — DISMISSED** [review]

**Comment by carlocab** [comment]
Oops. Have to authenticate. Never mind, `curl | jq` it is.

**Comment by carlocab** [comment]
Applying the `CI-long-timeout` label here errors as expected:
https://github.com/Homebrew/homebrew-core/runs/4884155117?check_suite_focus=true#step:3:111

**Pull request closed without merging**